client/src/components/commonComponents/TabsCheckout/FormModel/validationSchema.js

```diff
@@ -0,0 +1,21 @@
+import * as Yup from 'yup';
+import checkoutFormModel from './checkoutFormModel';
+const {
+  formField: { fullName, phone, email, zipCode },
+} = checkoutFormModel;
+const phoneRegExp = /^((\\+[1-9]{1,4}[ \\-]*)|(\\([0-9]{2,3}\\)[ \\-]*)|([0-9]{2,4})[ \\-]*)*?[0-9]{3,4}?[ \\-]*[0-9]{3,4}?$/;
+
+export default [
+  Yup.object().shape({
```

Comment:
Maybe you can share these schemas with the backend to apply the same validations there

Reply:
yeah, we will do that.